LICENSE

```diff
@@ -1,4 +1,4 @@
-Copyright 2010-2019 Urban Airship and Contributors
+Copyright 2010-2020 Airship and Contributors
```

Comment:
FWIW, looks like the date needs to be updated in the iOS SDK. Haven't checked any of the others.

Reply:
open a ticket please